lib/cocoapods/validator.rb

```diff
@@ -450,7 +450,7 @@ def validate_screenshots(spec)
     # Performs validations related to the `social_media_url` attribute.
     #
     def validate_social_media_url(spec)
-      validate_url(spec.social_media_url) if spec.social_media_url
+      validate_url(spec.social_media_url, 'Not a Browser') if spec.social_media_url
```

Comment:
should this be 'CocoaPods' or is this standard practice for unknown/placeholder user agents?